pokete.py

```diff
@@ -1074,6 +1074,26 @@ def __call__(self, pokes):
             self.rem_c_obs()
 
 
+class Help(About):
+    def __init__(self, map):
+        self.map = map
+        self.help_text = """
+Controls:
+'w':up, 'a':left, 
+'s':down, 'd':right,
+'e':menu
+
+When walking into the hight grass (';') you may get attacked
+by wild Poketes, those can be killed or weakened and caught.
+NPCs will talk to you when walking up to them.
+For more information about how to play this game, check out
+HowToPlay.md
```

Comment:
Consider making this reference a link to this repo. People who don't know all the installation files or don't have a markdown viewer installed may not understand the file or what your reference means. 

Tipp: maybe look into GitHub pages to shorten the links. It is pretty easy and works with your wiki and the other files nicely 😉.

Reply:
That's true, I will look into that.

Reply:
Added the shortened link